.gitignore

```diff
@@ -4,6 +4,11 @@
 # Log file
 *.log
 testIgnore/
```

Comment:
Please delete, if it's not needed anymore

Reply:
Done